main.js

```diff
@@ -0,0 +1,13 @@
+const allPosts = document.querySelector(`li`);
+allPosts.remove();
+
+const span = document.querySelector(`span`);
+span.remove();
+
+const lastPost = document.querySelector(`h3`);
+lastPost.remove();
```

Comment:
There are multiple `h3` elements in the html file. The instructions specify to remove the entire last post.